prodigy.el

```diff
@@ -1461,6 +1473,31 @@ beginning of the line."
   (font-lock-mode 1)
   (use-local-map prodigy-view-mode-map))
 
+
+;;;###autoload
+(defun prodigy-auto-start ()
+  "Start services whose property `auto-start' is set to `t'."
+  (interactive)
+  (let ((saved-buffer (current-buffer))
+	(buffer-p (prodigy-buffer))
+        (buffer (get-buffer-create prodigy-buffer-name)))
```

Comment:
@seblemaguer 	For example here.  This should be aligned.  The `let` form body should be 2 spaces indented.

Use `indent-region` on the defun to use the default emacs lisp formatting rules, it is always correct.

Reply:
ok thanks a lot :)

I update the pull request and tell you when I finished with it.